views/pages/ViewChartPage/index.jsx

```diff
@@ -108,7 +113,32 @@ const ViewChartPage = () => {
       barsToDisplay: 10,
     },
   }
+  const handleChangePage = (_, newPage) => setPage(newPage)
 
+  const handleChangeRowsPerPage = (event) => {
+    setRowsPerPage(parseInt(event.target.value, 10))
+    setPage(0)
+  }
+  const comparator = (a, b) => {
+    if (b[sortBy] < a[sortBy]) {
+      return sortDirection === SORT_DIRECTION.ASC ? 1 : -1
+    }
+    if (b[sortBy] > a[sortBy]) {
+      return sortDirection === SORT_DIRECTION.ASC ? -1 : 1
+    }
+    return 0
+  }
+  const stableSort = (array, comparator) => {
+    const stabilizedThis = array.map((el, index) => [el, index])
+    stabilizedThis.sort((a, b) => {
+      const order = comparator(a[0], b[0])
+      if (order !== 0) {
+        return order
+      }
+      return a[1] - b[1]
+    })
```

Comment:
This is to add a sort to older browsers. Since we don't support older browsers I simplified it.
[069c7f5](https://github.com/AMP-SCZ/dpdash/pull/695/commits/069c7f5f8b96d1d5ac74f8a4a3120484b1435b63)